app/src/styling/pages/_landing.scss

```diff
@@ -0,0 +1,88 @@
+$body-bg: #FFDA7C;
+$purple: #6A4C93;
+
+*{
```

Comment:
this general styling shouldnt be here, this should go to the _base.scss file since its something that will be applied to the full website